src/main/java/eu/arrowhead/blacklist/api/http/ManagementAPI.java

```diff
@@ -56,6 +67,58 @@ public class ManagementAPI {
 	//=================================================================================================
 	// methods
 
+
+	// LOG
+
+	//-------------------------------------------------------------------------------------------------
```

Comment:
As I mentioned before these are refactored into a different service. So you need a new class for that (GeneralManagementAPI). See the actual version of SR.